src/engine/GraphStoreProtocol.h

```diff
@@ -81,6 +89,9 @@ class GraphStoreProtocol {
   CPP_template_2(typename RequestT)(
       requires ad_utility::httpUtils::HttpRequest<RequestT>) static ParsedQuery
       transformPost(const RequestT& rawRequest, const GraphOrDefault& graph) {
+    if (rawRequest.body().empty()) {
+      throw HttpError(boost::beast::http::status::no_content);
```

Comment:
Can you enrich all of those with a human-readable message, s.t. we have both, the proper status code, and something to fix here?